### PR TITLE
new feature: add privacy-router for sf-lr-mix

### PR DESCRIPTION
**Pull request by strongman1995**
**Type of change**

- [ ] Add new papers (Please tell us why you think this paper is awesome!)
- [ ] Fix the category of an existing paper/papers (Please tell us the reasons)
- [x] Add a new tool/primitive/application with a new markdown page (Thank you! Also, please tell us more about this awesome thing!)

**Description**

> Please include a summary of the change and the relevant motivation.
Supply a new component "Privacy-Router" for enabling mutual conversion between homomorphically encrypted data and secret-shared data.

### CODE REVIEW

File: router/demo/he2ss/he_rc_03.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+from anyconn_core import AppConfig
```
[the remark this answers is not in the document]
Comment (strongman1995) [reply router/demo/he2ss/he_rc_03.py:16]: 已添加

File: router/demo/.DS_Store
Comment (usafchn) [inline router/demo/.DS_Store:1]: 这个文件删掉，可以把 .DS_Store 加到 .gitignore 里面

File: router/img/.DS_Store
Comment (usafchn) [inline router/img/.DS_Store:1]: 同上

File: router/demo/.DS_Store
Comment (strongman1995) [reply router/demo/.DS_Store:1]: 已删除

File: router/img/.DS_Store
Comment (strongman1995) [reply router/img/.DS_Store:1]: 已删除

**Review by usafchn — APPROVED** [review]

**Comment by usafchn** [comment]
CLA 签署信息 https://github.com/secretflow/secretflow/pull/753

**Pull request merged**